src/System.Threading/tests/InterlockedTests.netcoreapp.cs

```diff
@@ -97,7 +98,9 @@ private LockCookie EnterSlow()
             }
         }
 
-        [Fact]
+        public static bool IsNotArm64 => RuntimeInformation.ProcessArchitecture != Architecture.Arm64;
```

Comment:
Could you please move this to corefx\src\CoreFx.Private.TestUtilities\src\System\PlatformDetection.cs  ? With the others.

Eg that already has
```
        public static bool IsArmProcess => RuntimeInformation.ProcessArchitecture == Architecture.Arm;
        public static bool IsNotArmProcess => !IsArmProcess;
```